test/alternator/test_describe_table.py

```diff
@@ -114,10 +114,10 @@ def test_describe_table_item_count(test_table):
 
 # Similar test for estimated size in bytes - TableSizeBytes - which again,
 # may reflect the size as long as six hours ago.
-@pytest.mark.xfail(reason="DescribeTable does not return table size")
 def test_describe_table_size(test_table):
     got = test_table.meta.client.describe_table(TableName=test_table.name)['Table']
     assert 'TableSizeBytes' in got
+    assert got['TableSizeBytes'] >= 0
```

Comment:
I don't see any point to checking that it's >= 0. It's not like we're expecting it can ever be negative... Checking for ">= 0" also allows the unfixed code (which returned 0) to pass this test, so it's not a test for your patch.